devel/CUPS/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                APPLE-CUPS
+version             2.3.3
+revision            0
+categories          devel
+platforms           darwin
+supported_archs     ppc i386 x86_64 arm64
+license             Apache-2
+license_noconflict  autoconf
+maintainers         nomaintainer
+platforms           darwin
+description         The standards-based, open source printing system developed by Apple for iOS®, iPadOS®, and macOS®. \
+                    CUPS uses IPP Everywhere™ to support printing to local and network printers.
+long_description    CUPS is a standards-based, open source printing system developed by Apple Inc. \
+                    for macOS® and other UNIX®-like operating systems.  CUPS uses the Internet \
+                    Printing Protocol ("IPP") and provides System V and Berkeley command-line \
+                    interfaces, a web interface, and a C API to manage printers and print jobs.  It \
+                    supports printing to both local (parallel, serial, USB) and networked printers, \
+                    and printers can be shared from one computer to another, even over the Internet! \
+                    Internally, CUPS uses PostScript Printer Description ("PPD") files to describe \
+                    printer capabilities and features and a wide variety of generic and device- \
+                    specific programs to convert and print many types of files.  Sample drivers are \
+                    included with CUPS to support many Dymo, EPSON, HP, Intellitech, OKIDATA, and \
+                    Zebra printers.  Many more drivers are available online and (in some cases) on \
+                    the driver CD-ROM that came with your printer.
+homepage            https://www.cups.org/
+fetch.type          git
+git.url             https://github.com/apple/cups.git
+depends_build       port:autoconf
+configure.cmd       ./configure --prefix=${prefix} --with-archflags="${configure.universal_cxxflags}"
+build.type          default
+build.target        ENTER
```

Comment:
what is this target? I don't see it in the Makefile, why does the default not work?